documentation/samples/packs/todoist/formula.ts

```diff
@@ -20,17 +20,17 @@ const TaskSchema = coda.makeObjectSchema({
     },
     taskId: {
       description: "The ID of the task.",
-      type: coda.ValueType.Number,
+      type: coda.ValueType.String,
       required: true,
     },
   },
   displayProperty: "name",
   idProperty: "taskId",
 });
 
-// Formula that looks up rich metadata about a task given it's URL.
+// Formula that looks up rich metadata about a task given it's ID.
 pack.addFormula({
-  name: "GetTaskById",
+  name: "Task",
```

Comment:
Is this okay that it's a dup formula name

Reply:
Ah I think I see that these samples are structured as separate files and then full is everything together, disregard

Reply:
> h I think I see that these samples are structured as separate files and then full is everything together

Yes, that's correct. The shorter ones are used to highlight specific concepts.